cmd/collectors/zapi/plugins/sensor/sensor.go

```diff
@@ -338,5 +344,17 @@ func (my *Sensor) calculateEnvironmentMetrics(data *matrix.Matrix) ([]*matrix.Ma
 			}
 		}
 	}
+
+	if len(whrSensors) > 0 {
+		var whrSensorsStr string
+		for _, v := range whrSensors {
+			if v != nil {
```

Comment:
i don't think v can be nil so check not needed. Alternatively you could change `whrSensors` to `make(map[string]sensorValue)` too